docs/cloud/manage/role-based-access.md

```diff
@@ -0,0 +1,139 @@
+| Remove Nodes from Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+
+##### Notifications Management
+
+| **Functionality** | **Administrator** | **Manager** | **Troubleshooter** | **Observer** | **Billing** | Notes |
+| :-- | :--: | :--: | :--: | :--: | :--: | :-- |
+| See all configured notifications on a Space  | :heavy_check_mark: | - | - | - | - | |
+| Add new configuration | :heavy_check_mark: | - | - | - | - | |
+| Enable/Disable configuration | :heavy_check_mark: | - | - | - | - |  |
+| Edit configuration | :heavy_check_mark: | - | - | - | - | Some exceptions apply depending on [service level](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md#available-actions-per-notification-methods-based-on-service-level) |
+| Delete configuration | :heavy_check_mark: | - | - | - | - | |
+| Edit personal level notification settings | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [Manage user notification settings](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md#manage-user-notification-settings) |
+
+Notes:
+* Enable, Edit and Add actions over specific notification methods will only be allowed if your plan has access to those ([service classification](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/notifications.mdx#service-classification))
+
+##### Dashboards
+
+| **Functionality** | **Administrator** | **Manager** | **Troubleshooter** | **Observer** | **Billing** |
+| :-- | :--: | :--: | :--: | :--: | :--: |
+| See all dashboards in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - |
+| Add new dashboard to Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - |
+| Edit any dashboard in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | - |
+| Edit own dashboard in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - |
+| Delete any dashboard in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | - |
+| Delete own dashboard in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - |
+
+##### Functions
+
+| **Functionality** | **Administrator** | **Manager** | **Troubleshooter** | **Observer** | **Billing** | Notes |
+| :-- | :--: | :--: | :--: | :--: | :--: | :-- |
+| See all functions in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - |
+| Run any function in Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - |
+| Run read-only function in Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | There isn't any function on this category yet, so subject to change. |
```

Comment:
Our first function is actually a read-only function. I would suggest to remove the comment

Reply:
new PR https://github.com/netdata/netdata/pull/14444